terraform/outputs.tf

```
@@ -0,0 +1,13 @@
+output "app_name" {
+  value = juju_application.admission_webhook.name
+}
+
+output "provides" {
+  value = {
+    pod_defaults = "pod-defaults",
+  }
+}
+
+output "requires" {
+  value = {}
```

We also need the logging relation, right? Found [here](https://github.com/canonical/admission-webhook-operator/blob/f8f8e460ba778eafe6ef0e419b531be92fd34ebb/metadata.yaml#L24).
```suggestion
  value = {
      logging = "logging"
  }
```

---

Good catch, updated!